pandas/tests/frame/test_stack_unstack.py

```diff
@@ -2181,9 +2202,11 @@ def test_stack_nullable_dtype(self):
         df[df.columns[0]] = df[df.columns[0]].astype(pd.Float64Dtype())
         result = df.stack("station")
 
-        # TODO(EA2D): we get object dtype because DataFrame.values can't
-        #  be an EA
-        expected = df.astype(object).stack("station")
+        expected = (
```

Comment:
For this `expected` could you construct without the `astype(object)` e.g. `pd.DataFrame(...)`?

Reply:
sure, updated.